admin/partials/nginx-helper-general-options.php

```diff
@@ -287,9 +293,14 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>%5$s<strong>%6$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'post', 'nginx-helper' ), esc_html__( ' (or page/custom post) is ', 'nginx-helper' ), esc_html__( 'modified', 'nginx-helper' ), esc_html__( ' or ', 'nginx-helper' ), esc_html__( 'added', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'post', 'nginx-helper' ),
+												esc_html__( ' (or page/custom post) is ', 'nginx-helper' ),
+												esc_html__( 'modified', 'nginx-helper' ),
+												esc_html__( ' or ', 'nginx-helper' ),
+												esc_html__( 'added', 'nginx-helper' )
```

Comment:
This is not correct way to make small string translatable. We need to allow entire string translatable.